docs/containers/docker-compose.md

```diff
@@ -0,0 +1,217 @@
+
+The Docker extension adds the following files to your workspace:
+
+- *docker-compose.yml* – Brings up the containers as expected in production.
+- *docker-compose.debug.yml* – Provides a simplified mode for starting that enables the debugger. This is not available with Python. For Python, see [How to debug your app with Gunicorn](/docs/containers/debug-python.md#how-to-debug-your-app-with-gunicorn).
+
+![Screenshot of project with docker-compose files](./images/compose/docker-compose-files.png)
+
+The VS Code Docker extension generate the files, but you also need to understand them and configure them for your scenario. Configuration includes setting up volume mounts (your own folders shared with the container that your app needs), and any setup or configuration that's needed for the container that hosts your app. You can then use the **Docker Compose Up** command (right-click on the *docker-compose.yml* file to find the menu item for this, or find the command in the Command Palette). You can also use the `docker-compose up` command from the command prompt or terminal window in VS Code to start the containers. Refer to the [Docker Compose docs](https://docs.docker.com/compose/up) about how to configure the Docker Compose behavior and what command-line options are available.
+
+![Screenshot of Docker Compose Up command](images/compose/compose-up.png)
+
+You specify the ports that your app uses in the json configuration files in the `.vscode` folder, `launch.json` and `tasks.json`. To view or change the port mappings in the config file, inspect the container in the Docker tab to open the json configuration file, and look for `Ports` (which is the port in the container) with the child item `HostPort`(port this maps to on the host machine).
+
+```json
+    "NetworkSettings": {
+        "Bridge": "",
+        "SandboxID": "7d1ea3e06002a76bf13cb58bf59a99fc5b8e6a3b9db82af86b0e078e600b51ce",
+        "HairpinMode": false,
+        "LinkLocalIPv6Address": "",
+        "LinkLocalIPv6PrefixLen": 0,
+        "Ports": {
+            "443/tcp": null,
+            "80/tcp": [
+                {
+                    "HostIp": "0.0.0.0",
+                    "HostPort": "32768"
+                }
+            ]
+        },
+        //...
+    }
+```
+
+>[!TIP] When using Docker Compose, don't specify a host port. Instead, let the Docker pick a random available port to automatically avoid port conflict issues.
+
+## Add new containers to your projects
+
+If you want to add a second app or service, modify the `docker-compose.yml` file to add the new app. Typically, you can cut and paste the existing section and change the names as appropriate for the new app.
+
+You can run the **Add Docker Files to Workspace** command again to generate the `Dockerfile` for a new app. There's one `docker-compose.yml` and one `docker-compose.debug.yml` file for project for .NET Core and Python, or one per package.json for Node.js.
```

Comment:
See my comment regarding upcoming **Add Docker Compose Files to Workspace...** command above. One would use "Add Docker Files..." if the new project has no Docker files yet, but "Add Docker Compose Files..." if just the Compose part is missing